tests/codegen/autodiff/inline.rs

```diff
@@ -0,0 +1,20 @@
+//@ compile-flags: -Zautodiff=Enable -Zautodiff=NoPostopt -C opt-level=3  -Clto=fat
+//@ no-prefer-dynamic
+//@ needs-enzyme
+
+#![feature(autodiff)]
+
+use std::autodiff::autodiff;
+
+#[autodiff(d_square, Reverse, Duplicated, Active)]
+fn square(x: &f64) -> f64 {
+    x * x
+}
+// CHECK: ; Function Attrs: alwaysinline noinline
+// CHECK-NEXT: declare double @__enzyme_autodiff_ZN6inline8d_square17h021c74e92c259cdeE(...) local_unnamed_addr #8
```

Comment:
```suggestion
// CHECK-NOT: noinline
// CHECK-NEXT: declare double @__enzyme_autodiff_ZN6inline8d_square17h021c74e92c259cdeE(...) local_unnamed_addr #8
```

Reply:
Lit tests don't care about extra text at the end of a test line, so just removing form the CHECK isn't enough, it would still pass.